docker/VERSION

```diff
@@ -1,2 +1,2 @@
-DOCKERFILE_VERSION=0.6
-GOOGLE_FLUENTD_VERSION=1.6.0-1
+DOCKERFILE_VERSION=0.7
```

Comment:
[Optional] I wonder if it makes sense to bump the Dockerfile version if all we're changing is the embedded package. The image tag will already reflect that — should we save the Dockerfile version bump for actual changes in the Dockerfile itself?

Reply:
Ultimately we probably want to get rid of the package version bit? Once we have a clear way to track down package versions. Today they are there more for informational purpose.